editor/src/java/com/defold/editor/Start.java

```diff
@@ -151,6 +187,30 @@ public void start(Stage primaryStage) throws Exception {
             logger.error("failed start", t);
             throw t;
         }
+
+        try {
+            if (Taskbar.isTaskbarSupported() && Taskbar.getTaskbar().isSupported(Taskbar.Feature.MENU) && isPathsAvailable()) {
+                Taskbar taskbar = Taskbar.getTaskbar();
+                PopupMenu dockMenu = new PopupMenu();
+                MenuItem dockMenuItem = new MenuItem("New window");
+                dockMenuItem.addActionListener(new ActionListener() {
+                    @Override
+                    public void actionPerformed(ActionEvent e) {
+                        try {
+                            newWindow();
+                        } catch (Throwable t) {
+                            logger.error("Can't open new window", t);
+                        }
+                    }
+                });
+                dockMenu.add(dockMenuItem);
+                taskbar.setMenu(dockMenu);
+            }
+        } catch (final UnsupportedOperationException e) {
+            logger.error("The OS does not support taskbar menu", e);
```

Comment:
Can this even happen after the checks we do? If so, it should probably be logged as info, not an error.